Bio/ExPASy/Enzyme.py

```diff
@@ -55,7 +55,7 @@ def read(handle):
 
 
 class Record(dict):
-    """\
+    r"""\
```

Comment:
@peterjc I have made this and the other fixes and am re-running the test now. We shall soon see!